app/site/controllers.py

```diff
@@ -52,6 +51,7 @@ def logout():
       302:
         description: Load the Home Page
     """
+    g.current_user = None
     resp = make_response(render_template("logout.html", title='Logout'), 200)
     resp.set_cookie('jwt', '', expires=0)
```

Comment:
Also, I guess there is no way to expire the JWT (that's how JWT's work right ...?)